### ISSUE TITLE
On seek video get paused on Fire TV

### ISSUE THREAD

**Issue by 0x6d61726b** [jellyfin/jellyfin-androidtv#3154]
### Describe the bug

When we seek in a video forward/backward with the Fire TV remote, the seek bar appears and the video playback pauses until the play button is pressed again using both, ExoPlayer and LibVLC. This is very annoying and leads to massive family complaints.
This issue is directly related to the closed issue #2399 (which was auto-closed and I am unable to re-open) where I provided additional information.

Is this an issue related to a misconfiguration on our clients I did during setup/configuration?
If not, is this issue possibly fixed with #2826 and is there a test version available?

In general Jellyfin and Jellyfin-AndroidTV is a great software I would like to continue using it for our family. I would also like to provide some support. Unfortunately, I have no experience with Java programming language, nor Android programming.

### Application version

0.15.12

### Device information

Amazon AFTR (FireTV Cube 2nd Gen)

### Android version

Fire OS 7.6.6.4

### Jellyfin server version

10.8.10.0

**Comment by nielsvanvelzen** [comment]
This is intended behavior (it was programmed to work this way) but we should definitely change it. Unfortunately this behavior is not defined by us but a third-party library we use so we cannot change it until we rewrite the player user interface.

**Comment by hmbqzidixiwc** [comment]
I find this to be an annoyance as well but I do understand it is not under the control of jellyfin.

As a workaround I do have a suggestion, would it be possible to detect seeking then re-instate the last playback.

If a seek occurs, check previous playback state, on seek completed restore playback state.

So if it were seeking when paused then pause, if it were playing then resume playing.

Or jellyfin is completely out of the loop during playback and none of that is available?

**Comment by 0x6d61726b** [comment]
keep open: issue still persists

**Comment by 0x6d61726b** [comment]
keep open: issue still persists (checked with v0.16.11)

**Comment by sfyfq** [comment]
I found a hack/workaround to automatically resume playing after the user stops scrubbing. This works by scheduling a Runnable when `onSeekPositionChanged` is called, which when executed, emits an emulated KEYCODE_ENTER to trigger `onSeekFinished(false)`. The Runnable gets rescheduled if the user scrubs the seekbar again before the timeout fires. This implementation also preserves the behavior of returning to the last position before scrubbing is started if the user presses "BACK" button at any time. In order to change the code inside `onSeekPositionChanged`, I shadowed the package-private class `SeekUiClient` in PlaybackTransportControlGlue.java by copying it over to CustomPlaybackTransportControlGlue.java. I tested this code on my TV and it works quite nicely.

https://github.com/sfyfq/jellyfin-androidtv/tree/3febd49ab125904f5f84a8829c8bbb71139d870a

**Comment by wolfdael** [comment]
@nielsvanvelzen Is this issue part of the playback rewrite and fixed later on?

Now that subtitles issue is completely fixed, this issue is only what, which is really annoying and frustrating. Please find a logic to fix this issue or use above comment as solution.

**Comment by Pentaphon** [comment]
This seems like it could be changed in the latest media3 library, no?

**Comment by SN0WTR00PER** [comment]
Can we expect that bug being really fix or an option to allow us to select the behavior when using the seek bar?

Its quite a basic thing and this need to be fix.

**Comment by thornbill** [comment]
> Can we expect that bug being really fix or an option to allow us to select the behavior when using the seek bar?

[No](https://jellyfin.org/docs/general/faq#why-dont-you-support-my-favorite-client-or-feature)

> Its quite a basic thing and this need to be fix.

PRs are welcome!

**Comment by mschumacher69** [comment]
Why the absolute "No"?? @nielsvanvelzen himself said "we should definitely change it", so it is being considered...

**Comment by SN0WTR00PER** [comment]
> Did you read what was linked?
> 
> > Jellyfin is an entirely volunteer-driven project, so until a developer is able and willing to implement a feature, it likely will not be done.
> 
> So no you cannot "expect" anyone to do anything for you. 

You need to relax a bit for sure.

I don't expect anything done for me but for all the supporters and users of Jellyfin.

Yes, I've read the post and yes, I knows what it implies to do, but It's not impossible so why you answer for every one....  If you don't have the answer, don't reply... If you are stressed by your life, go see your psy, if your sad, go cry to your mom.  But don't pollute the forum with your scrappy answer pre-cooked by your own personal...

**Comment by mschumacher69** [comment]
Yes it is a volunteer-driven project, that's actually why I expect it to be done at some point because any developer that is willing to do it can do it. If you are not willing to do it, it doesn't mean no one else is willing to do it, so you cannot answer with an absolute No on behalf of all developers.

**Comment by wolfdael** [comment]
> Yes it is a volunteer-driven project, that's why I actually expect it to be done at some point because any developer that is willing to do it can do it. If you are not willing to do it, it doesn't mean no one else is willing to do it, so you cannot answer with an absolute No on behalf of all developers.

True that. This is is most horribly run open source project I have come across. And even the current devs can't handle constructive criticism. But they are ready to reply like teenagers.
They will say prs are welcome but when we take time and do a pr they won't merge it. Hell, they won't even clearly tell why or how to fix. Worst team.

https://github.com/jellyfin/jellyfin/pull/12843

**Comment by preetmyob** [comment]
Since this is still an issue in a third party library, could anyone point me at the library and I'll have a look at the code there, please?
